src/main/java/org/mapstruct/intellij/expression/JavaExpressionInjector.java

```diff
@@ -49,7 +49,7 @@
  */
 public class JavaExpressionInjector implements MultiHostInjector {
 
-    private static final Pattern JAVA_EXPRESSION = Pattern.compile( "\"java\\(.*\\)\"" );
+    public static final Pattern JAVA_EXPRESSION = Pattern.compile( "\" *java\\(.*\\) *\"" );
```

Comment:
Perhaps we need to do something like we are doing in core. 

See 

https://github.com/mapstruct/mapstruct/blob/fa857e9ff46b5bb3e5a6c2c8b1c56009944f5ef3/processor/src/main/java/org/mapstruct/ap/internal/model/source/MappingOptions.java#L40

Reply:
I agree. The current implementation is missing the `Pattern.DOTALL` and does not match all whitespace characters.